README.md

```diff
@@ -50,7 +68,29 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+This code doesn't make sense to me really, as written. I don't understand why you would call a "add_all_users" function on a GET request, so I've rewritten it below. Something like:
+
+```
+UserApp.get_all_users = function() {
+  $.ajax({
+    url: 'http://ig-hacker-news.herokuapp.com/users',
+    type: 'GET',
+  })
+  .done(function(data) {
+    console.log("success!")
+  });
+  UserApp.show_all_users(data);
+};
+
+UserApp.show_all_users = function(data) {
+  for (i = 0; i < results.length; i++){
+    $('#users').append('<li>' + results[i].name + '</li>');
+  };};
+```
+
+##### Original code:
```

Comment:
Sure, `show_all_users` makes more sense than `add_all_users`, but that's not the biggest problem. The `UserApp.add_all_users(data);` line needs to be inside of the `.done()` callback in order to have access to `data`.